app/build.gradle

```diff
@@ -138,9 +138,19 @@ android {
         // WebView
         ////////////////////////////////////////////////////////////////////////////////////////////
 
-        // Release
+        androidTest {
+            res.srcDir('src/androidTest/assets')
+            java.srcDirs += ['src/androidTest/java']
+        }
```

Comment:
This shouldn't be needed. The androidTest directory is part of the default configuration of every Android project. Do you really need to define this here?

Reply:
I put it as just as a precaution, but probably it's not needed.